terra-jupyter-aou/Dockerfile

```diff
@@ -279,5 +279,11 @@ RUN pip install --force-reinstall -v "Jinja2==3.0.3"
 
 ENV PATH="$PATH:/usr/local/RepeatMasker"
 
+# Turn off beatrix-jupyterlab and bq_stats plugins, they don't activate cleanly in our version
+# of jupyter and they're google-managed packages we don't have access to older versions of.
+# Retain two plugins: google.cloud.bigquery and sql
+RUN sed -i "s/^\( *'beatrix_jupyterlab',\)/#\1/g" /etc/ipython/ipython_kernel_config.py
+RUN sed -i "s/^\( *'bq_stats',\)/#\1/g" /etc/ipython/ipython_kernel_config.py
```

Comment:
This worries me a little because I'm not sure whether anyone is using these extensions. They are google-managed Python packages and we can't install versions that are compatible with our older notebook version, so I'm turning them off. Same deal in `terra-jupyter-python`.

Reply:
I am comfortable making that decision. Once we update to a newer version of notebooks we can add them back if we think we need it.